test/Microsoft.TemplateEngine.TestHelper/TestHost.cs

```diff
@@ -122,10 +114,5 @@ public void LogDiagnosticMessage(string message, string category, params string[
         {
             Console.WriteLine($"[{HostIdentifier}][{nameof(LogDiagnosticMessage)}][{category}] {message} (details: {string.Join(";", details)})");
         }
-
```

Comment:
probably we should kill it also from `DefaultEngineHost`, in case we want to remove it for interface.